[ci:component:github.com/gardener/gardener:v0.35.2->v1.0.2]

gardener-robot-ci-1 opened the pull request:
*Release Notes*:
```noteworthy operator
Upgrade Gardener to `v1.0.2`
```
```improvement operator
Upgrade Gardener extensions to `v1.3.0`. Since each extension now has its own repository, they will likely have independent releases too in the future.
```
```improvement operator
Upgrade Gardener dashboard to `1.37.1`
```
```improvement operator
Fix an issue with the naming of bootstrap tokens which could cause the `gardencontent/seeds/seeds` component to fail depending on the configured shoot/seed name.
```

Commit:
Upgrade github.com/gardener/gardener

from v0.35.2 to v1.0.0

Diaphteiros committed:
extension repo split + update + dashboard update

Diaphteiros commented:
/test --cloudprovider=aws --cloudprovider=gcp --cloudprovider=azure --kubernetes-version=1.17.1

Diaphteiros committed:
add missing regions to azure cloudprofile

Diaphteiros commented:
/test --cloudprovider=aws --cloudprovider=gcp --cloudprovider=azure --kubernetes-version=1.17.1

Diaphteiros commented:
/test --host-cloudprovider=aws--cloudprovider=aws --cloudprovider=gcp --cloudprovider=azure --kubernetes-version=1.17.1

Diaphteiros commented:
/test --host-cloudprovider=aws --cloudprovider=aws --cloudprovider=gcp --cloudprovider=azure --kubernetes-version=1.17.1

Diaphteiros committed:
fix bootstrap token naming issue

There was an issue with the naming of the bootstrap tokens for the gardenlet which could result in garden-setup being unable to create the seed depending on its name.

Diaphteiros commented:
/test --host-cloudprovider=aws --cloudprovider=aws --cloudprovider=gcp --cloudprovider=azure --kubernetes-version=1.17.1

Diaphteiros commented:
/test --host-cloudprovider=gcp --cloudprovider=aws --cloudprovider=gcp --cloudprovider=azure --kubernetes-version=1.17.1

Diaphteiros committed:
Upgrade Gardener to v1.0.2

Diaphteiros commented:
/test --host-cloudprovider=gcp --cloudprovider=aws --cloudprovider=gcp --cloudprovider=azure --kubernetes-version=1.17.1
